src/components/Table.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { useMemo } from 'react';
+import { Person } from '../types';
+
+const PersonLink = ({ person }: { person: Person }) => {
+  return (
+    <Link
+      to={`/people/${person.slug}`}
+      className={cn({ 'has-text-danger': person.sex === 'f' })}
+    >
+      {person.name}
+    </Link>
+  );
+};
+
+const Table = ({ people }: { people: Person[] }) => {
+  const param = useParams();
+
+  const computedPeople = useMemo(
```

Comment:
I have my doubts regarding this `useMemo` usage. 

Sure, it looks like we are making some sort of a complex computation here: map + find + find. 

However, from the other side, you are passing an array of people as a dependancy here. `useMemo` and `useEffect` use shallow comparison to determine if dependencies have changed. If the object or array is recreated on every render (even if its content doesn't change), it will cause the memoized value to be recomputed unnecessarily.